### DISCUSSION TITLE
Show Previews - Side by side my own components

### DISCUSSION THREAD

**Discussion by danielnc** [primer/view_components#1313] [Q&A]
Is there a way to show primer view component previews side-by-side with my own components using lookbook?

I want to use that as the source of truth to document all components that exists on the app, but I cant find a way to "import" existing primer components without copying them to the root of my app

I want to leverage the existing preview components and just add them to my own app

**Comment by jonrohan** [comment]
That's a good question. In theory it should be possible, in our own lookbook development.rb config, I point to a separate folder where the previews are stored.

https://github.com/primer/view_components/blob/6d8d2894121df80d78fb23829aa3ae8cafa19fcb/lookbook/config/environments/development.rb#L69

If your vendor folder is setup in such a way you might be able to include them that way? Otherwise, there might be something we need to do in the library that would allow adding PrimerViewComponent previews to the config.

**Reply by danielnc** [reply]
@jonrohan problem with going the vendor folder route, is that it differs from different dev machines and also staging machines, so we don't want to enforce for our team
It would be really nice if there was an easy way to just point to the previews folder without hard-coding vendor for all of our apps.

If someone else has another suggestion, that would be great, otherwise ill wait to see if there is something that can be done on the library itself for now

**Reply by joelhawksley** [reply]
@danielnc I'd open an issue on ViewComponent and/or Lookbook 😄

**Comment by jonrohan** [comment]
This change will support adding the previews to your own lookbook instance https://github.com/primer/view_components/pull/1404

**Reply by danielnc** [reply]
@jonrohan great to hear!! 🚀
